api/api/auth/views.py

```diff
@@ -51,7 +52,10 @@ def post(self, request):
 
             user = self._find_or_create_user(user_info)
             access_token = Token.find_or_create(user)
+            token_life = datetime.now(timezone.utc) - access_token.created_at
 
+            if(token_life.days >= 1):
```

Comment:
extract 1 in a constant variable. Maybe `TOKEN_LIFE_IN_DAYS`.

That way it would be much more readable and could be used somewhere else if needed.